sencha/index.py

```diff
@@ -102,19 +112,20 @@ def make_peptide_bloom_filter(
                     #  peptide kmer and add it directly
                     peptide_bloom_filter.add(hashed)
             except ValueError:
+                print("seq length!")
```

Comment:
can you instead do logger.info and have more info of why you would raise the error

Reply:
left in from testing. do we want to log this info?